packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -11,9 +11,15 @@ import {
 } from './SimpleStatistics.styles';
 
 const SimpleStatistic: FC = () => {
-    const { resultsData, explore } = useVisualizationContext();
+    const {
+        cartesianConfig: { dirtyEchartsConfig },
```

Comment:
@nathaliabj the big number shouldn't read from the cartesian config, there's a separate configuration for big numbers: https://github.com/lightdash/lightdash/blob/8d6ed06813b9d9415e173e4ae523b3a5f05a39f0/packages/common/src/types/savedCharts.ts#L78-L82

We need to extend that type by adding "label" to the configuration. Then the frontend will receive either a cartesian config or a big number config.

I was just looking through the code and the `VisualizationProvider` seems very focussed on `cartesian` configs, so that probably needs a few changes.